orangecontrib/text/widgets/utils/concurrent.py

```diff
@@ -94,7 +101,11 @@ def __init__(self, func):
     @Slot(object)
     def call(self, scope):
         instance, args, kwargs = scope
-        self.func.__get__(instance, type(instance))(*args, **kwargs)
+        try:
+            self.func.__get__(instance, type(instance))(*args, **kwargs)
+        except RuntimeError:
+            # C++ object wrapped by `obj` may be already destroyed
+            raise
```

Comment:
Why raising an exception here? I'm still get an error sometimes. It appears randomly and it is therefore hard to reproduce, but I do get it..

```
Traceback (most recent call last):
  File "/Users/vesna/orange3-text/orangecontrib/text/widgets/utils/concurrent.py", line 105, in call
    self.func.__get__(instance, type(instance))(*args, **kwargs)
  File "/Users/vesna/orange3-text/orangecontrib/text/widgets/owpreprocess.py", line 737, in on_progress
    self.progressBarSet(i, None)
  File "/Users/vesna/orange3/Orange/widgets/utils/progressbar.py", line 80, in progressBarSet
    .format(self.captionTitle, int(value), text))
RuntimeError: wrapped C/C++ object of type OWPreprocess has been deleted
```